<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>additional_window</class>
 <widget class="QDialog" name="additional_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить заметку</string>
  </property>
  <widget class="QLineEdit" name="note_title_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="note_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Заголовок заметки</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Текст заметки</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_content_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
